pages/common/wp.md

```diff
@@ -0,0 +1,24 @@
+# wp
+
+> The official command-line interface to manage WordPress instances.
+> More information: <https://wp-cli.org/>.
+
+- Describe the operating system, shell, PHP, and WP-CLI (`wp`) installation:
```

Comment:
Does this command print whole sentences? If not I think 
```suggestion
- Print information about the operating system, shell, PHP, and WP-CLI (`wp`) installation:
```
would be more fitting.